tests/testthat/test-methods.R

```diff
@@ -170,30 +170,30 @@ local({
   lints <- lint(text = "a", linters = test_linter())
   lint <- lints[[1L]]
 
-  wrapped_strings <- c(
-    "[test_linter]\n    The\n    quick\n    brown\n    fox\n    jumps\n    over\n    the\n    lazy\n    dog.",
-    "[test_linter]\n    The quick brown\n    fox jumps over\n    the lazy dog.",
-    "[test_linter] The\n    quick brown fox jumps over the lazy\n    dog.",
-    "[test_linter] The quick brown fox jumps over the lazy dog."
+  widths <- c(10L, 20L, 40L, 80L)
+  test_names <- paste0(": width = ", widths)
+
+  patrick::with_parameters_test_that(
+    "print.lint, print.lints support optional message wrapping",
+    {
+      withr::local_options(c(lintr.format_width = width))
+
+      expect_snapshot(print(lint))
+      expect_snapshot(print(lints))
```

Comment:
CMIIW but the `print(lint)` test looks pretty redundant vs. `print(lints)`, shall we just include the latter?